CICD.Tools.CatalogUpload.Lib/CatalogService/ContentType.cs

```diff
@@ -5,24 +5,31 @@
 	using System.Linq;
 
 	/// <summary>
-	/// As known by the Azure Artifact Uploader
+	/// As known by the Catalog Registration
 	/// </summary>
 	internal enum ArtifactContentType
 	{
-		Unknown = 0,
-		DmScript = 1,
-		Package = 2,
-		Visio = 3,
-		Function = 4,
-		Dashboard = 5,
-		CustomSolution = 6,
-		Example = 7,
-		CompanionFile = 8,
-		ProfileLoadScript = 9,
-		GQIOperator = 10,
-		ProcessActivity = 11,
-		DataGrabber = 12,
-		AdHocDataSource = 13,
+		automationscript,
```

Comment:
Instead of an enum, change this to a static class with consts or static readonly properties that are strings.

Reply:
Is there something wrong with using the .ToString() on an Enum? It's a well documented way to get the string value of an enum.